### ISSUE TITLE
Incorrect balance change shown on tx

### ISSUE THREAD

**Issue by what-name** [helius-labs/xray#104]
[TX example](https://xray.helius.xyz/2h7Va3hAjGrJtwYuPd8kkhf4kSawJvucXWui9kakJLVQHjryCRHCQzGrkPBa6u1c7uDCh8aCQ2XD4TdFdmtx4x3N/tx)

On a CodeCanvas purchase the sol balance change isn't displayed accurately. Here in total 10.02 SOL was sent from the wallet, but the dispalyed amount is 8.9.

The Helius description is also displaying an incorrect amount.

![image](https://user-images.githubusercontent.com/20590073/224206631-fafa6c50-4841-41ff-8070-7215e53addf0.png)

The "simple" parsed tx data from Proton seems correct tho.

```
{
      "account": "XXXXX",
      "changes": [
        {
          "amount": -10.02700408,
          "mint": "So11111111111111111111111111111111111111112"
        },
        {
          "amount": 1,
          "mint": "AMTC1Nhy4QHtRuAsWEx2jtwF8hKhQkksbJac64JDcCb3"
        }
}
```

**Comment by ssamkkim** [comment]
Yea the account changes are correct but the "actions" arent. Thanks for sharing this tx. Helius sometimes (from what I've found) with minting with sources from "SOLANA_PROGRAM_LIBRARY" usually don't get the mint price right, so when I originally parsed it I just chose the amount with the highest sol amount and selected that as the mint price, but seems like I need to add all of the transfers up instead.
